packages/twenty-front/src/modules/object-record/utils/sanitizeLinkRecordInput.ts

```diff
@@ -0,0 +1,11 @@
+export const sanitizeRecordLinks = (
+  recordObject: Record<string, any>,
+): Record<string, any> => {
+  return { ...recordObject, domainName: sanitizeLink(recordObject.domainName) };
+};
+const sanitizeLink = (link: string): string => {
+  let cleanedLink = link.replace(/^(ftp|http|https):\/\//, '');
+  cleanedLink = cleanedLink.replace(/\/+$/, '');
+  cleanedLink = cleanedLink.replace(/\/[^/]*$/, '');
+  return cleanedLink;
+};
```

Comment:
We also want to remove:
- `www.`
- search params after `?`

After testing `sanitizeLink`, here are some results I am obtaining:

<img width="308" alt="image" src="https://github.com/twentyhq/twenty/assets/3098428/ec6417f9-dca0-4cec-99c3-b142026feaba">

Maybe we could implement something like this?

```suggestion
const getUrlHostName = (url: string) =>
  URL.canParse(url) ? new URL(url).hostname.replace(/^www\./, '') : '';

const sanitizeLink = (url: string) =>
  getUrlHostName(url) || getUrlHostName(`https://${url}`);
```